docs/dataformat.rst

```diff
@@ -230,6 +237,62 @@ for these extractors. Currently, all intent classifiers make use of available re
     training data!
 
 
+Lookup Tables
+-------------
+Lookup tables in the form of external files or lists of elements may also be specified in the training data.  The externally supplied lookup tables must be in a newline-separated format.  For example, ``data/test/lookup_tables/plates.txt`` may contain
+
+.. include:: ../data/test/lookup_tables/plates.txt
+
+And can be loaded as:
+
+.. code-block:: json
+
+    {
+        "rasa_nlu_data": {
+            "lookup_tables": [
+                {
+                    "name": "plates",
+                    "elements": "data/test/lookup_tables/plates.txt"
+                }
+            ]
+        }
+    }
+
+Alternatively, lookup elements may be directly included as a list
+
+    {
+        "rasa_nlu_data": {
+            "lookup_tables": [
+                {
+                    "name": "plates",
+                    "elements": ["beans", "rice", "tacos", "cheese"]
+                }
+            ]
+        }
+    }
+
+In markdown data format, lookup table filenames are indicated by specifying a single file path directly after the lookup header.
```

Comment:
Hmm, generally all the markdown stuff is written under the markdown section, and the rest of the stuff is all in json. So I'd say just add an example of loading up a file in the MD training data file up there. you can just call it additional_currencies or something like that

Reply:
ok I cleared that up